File: Week1/LonelyInteger/src/main/java/MyCode.java
package main.java;

import java.util.List;

public class MyCode {


    public static int lonelyInteger(List<Integer> a) {
        int lonely = a.stream().filter( i-> a.stream().filter(it -> it == i ).count() == 1 ).toList().get(0);
        return lonely;
    }
}

File: Week1/LonelyInteger/src/main/kotlin/MyCode.kt
package main.kotlin

fun lonelyInteger(a: Array<Int>):Int {
    var lonely = 0
    for(i in a){
        if(a.count { it == i } == 1){
            lonely = i
        }
    }
    return lonely

}
fun main() {
    val n = readLine()!!.trim().toInt()

    val a = readLine()!!.trimEnd().split(" ").map{ it.toInt() }.toTypedArray()

    val result = lonelyInteger(a)

    println(result)
}